### ISSUE TITLE
AtomNet: A Deep Convolutional Neural Network for Bioactivity Prediction in Structure-based Drug Discovery

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#56]
https://arxiv.org/abs/1510.02855

**At a glance**: Related to virtual screening #45.  This approach does not use molecular fingerprints but rather the structural properties of ligand-protein target interactions.  The technique seems interesting, but the Methods lacks sufficient detail for me to fully understand what they're doing.

**Comment by agitter** [comment]
### Biology
- Want to predict molecular binding, specifically small molecule binding to a protein target.
- Instead of using molecular fingerprints of chemical compounds for input, take 3D grid of features of a compound-target co-complex.
- Molecular fingerprint approaches (e.g. #55) requires training data for the specific protein target, which they do not.  Their training data can span many targets but requires more detailed information about the atoms in the target binding site and the target structure.
- Study the DUD-E benchmark, an internal benchmark derived from ChEMBL, and verified inactive compounds.
- ChEMBL-based dataset has 78,904 actives, 2,367,120 decoys, and 290 targets.

### Computational aspects
- Related approaches (e.g. #55) use unordered data, haven't benefited from convolutions.
- Claim to be the first deep learning system to include structural information about a target to make predictions.  I'm not aware of any prior examples.
- Network is input -> four 3D convolutional layers -> two fully connected layers -> sigmoid output.  ReLU hidden units.
- Input features are derived from a co-complex of the compound and target.  Make a 20x20x20 grid over the 3D space.  Each cell in the grid contains some structural features, but they do not describe what these are specifically.
- Do not optimize hyper-parameters or network structure.
- Report logAUC to place more weight on top-ranked predictions.
- The evaluation is limited.  They compare to docking (Smina), an appropriate choice for a structure-based classifier, and clearly outperform it.  They also cite the performance of other docking systems on DUD-E.  But I'm not convinced that their performance is that great when #55 reports much better results (AUCs over 0.99 for all models) on DUD-E with a molecular fingerprint-based system.  Arguably the problem is harder here, but the reported AUCs and logAUCs could be given context.
- To interpret filters, they "examine the location where they maximally fire".
- No code

### Why include it in the review
- One of the few examples we have of 3D convolutions (#73 is another).  We could discuss strategies for dealing with unordered, 1D, 2D, and 3D data across domains.
- They take a rather different approach than the other virtual screening methods that use molecular fingerprints.
